docs/developer-docs/docs/build-an-agent/build-an-onchain-ai-agent/summary.md

```
@@ -0,0 +1,116 @@
+```
+cast call $ORDER "futureId()"
+cast call $ASYNC_PRECOMPILE "futureById(uint64)" $FUTURE_ID
+cast call $ORDER "validUntil()"
+cast call $ORDER "canExecute()"
+```
+
+## What's next?
+
+Please note that these Orders are just examples of what you can build with Agents. The sky is the limit! If you have an interesting idea, make a PR to the [example repo](https://github.com/warden-protocol/agent-kit-examples).
+
+Alternatively, you can reach out to us for any questions or feedback: developers@wardenprotocol.org
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)